web3/manager.py

```diff
@@ -314,25 +314,25 @@ async def coro_request(
 
     # -- persistent connection -- #
 
-    async def ws_send(self, method: RPCEndpoint, params: Any) -> RPCResponse:
+    async def send(self, method: RPCEndpoint, params: Any) -> RPCResponse:
         provider = cast(PersistentConnectionProvider, self._provider)
         request_func = await provider.request_func(
             cast("AsyncWeb3", self.w3), cast("MiddlewareOnion", self.middleware_onion)
         )
         self.logger.debug(
-            "Making request to open websocket connection - "
+            "Making request to open socket connection - "
             f"uri: {provider.endpoint_uri}, method: {method}"
         )
         response = await request_func(method, params)
-        return await self._process_ws_response(response)
+        return await self._process_response(response)
 
     def _persistent_message_stream(self) -> "_AsyncPersistentMessageStream":
         return _AsyncPersistentMessageStream(self)
 
-    async def _get_next_ws_message(self) -> Any:
-        return await self._ws_message_stream().__anext__()
+    async def _get_next_message(self) -> Any:
+        return await self._message_stream().__anext__()
 
-    async def _ws_message_stream(self) -> AsyncGenerator[RPCResponse, None]:
+    async def _message_stream(self) -> AsyncGenerator[RPCResponse, None]:
         if not isinstance(self._provider, PersistentConnectionProvider):
             raise TypeError(
                 "Only websocket providers that maintain an open, persistent connection "
```

Comment:
It looks like we're removing references to websockets, and either just using "socket" or "persistent connection". Should that happen here too?

Reply:
oop, yep. Nice catch!